corn/index.html

```diff
@@ -6,11 +6,67 @@
   <meta charset="utf-8">
   <title>Todo App</title>
   <link rel="stylesheet" href="styles.css" />
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;600;700&display=swap" rel="stylesheet">
   <!-- If you uncomment the following line the design MUST be dark -->
-  <!-- link rel="stylesheet" href="styles.black.css" / -->
+  <!--link rel="stylesheet" href="styles.black.css" /-->
 </head>
 
 <body>
-
+  <header>
+    <h1>
+      <time datetime="2020-03-09">March 9, 2020</time>
+      <p class="subtitle">5 incomplete, 5 completed</p>
+    </h1>
+  </header>
+  <hr />
+  <main>
+    <section>
+      <h2>Incomplete</h2>
+      <ul>
+        <li class="li">
```

Comment:
1. You can optimise your html by using class for `<ul>` element (for example, `class="incomplete_list"`) and have a rule for `ul.incomplete_list > li`
2. CSS classes MUST have semantic names as well. `li` class does not tell me anything.